BungeeCord-Patches/0054-Allow-to-disable-tablist-rewrite.patch

```diff
@@ -0,0 +1,72 @@
+From 9a4f155c89e3f83e81eae8189d62b6cca66fa046 Mon Sep 17 00:00:00 2001
+From: xDark <aleshkailyashevich@gmail.com>
+Date: Thu, 30 May 2019 18:54:09 +0300
+Subject: [PATCH] Allow to disable tablist rewrite
+
+
+diff --git a/api/src/main/java/net/md_5/bungee/api/ProxyConfig.java b/api/src/main/java/net/md_5/bungee/api/ProxyConfig.java
+index cbcf8a24..4ea7678e 100644
+--- a/api/src/main/java/net/md_5/bungee/api/ProxyConfig.java
++++ b/api/src/main/java/net/md_5/bungee/api/ProxyConfig.java
+@@ -239,4 +239,9 @@ public interface ProxyConfig
+      * @return Should we disable entity metadata rewriting?
+      */
+     boolean isDisableEntityMetadataRewrite();
++
++    /**
++     * @return Should we disable tab list rewriting?
```

Comment:
Javadoc comments shouldn't be like questions, they should describe things.

E.g `Whether tablist rewriting should be disabled or not` would fit better here.

Reply:
Well, I looked at the comment above.
Gonna change jdocs and add comments as soon as possible